lib/ansible/modules/source_control/gitlab_runner.py

```diff
@@ -311,7 +294,6 @@ def main():
     module = AnsibleModule(
         argument_spec=argument_spec,
         mutually_exclusive=[
-            ['api_url', 'url'],
             ['api_username', 'api_token'],
             ['api_password', 'api_token'],
         ],
```

Comment:
Remove `['api_url', 'url']` from `required_one_of`
Remove `['login_user', 'login_password']` from `required_together`

Reply:
@Lunik Done. Can you please take a look ? Thanks for the review.

Reply:
Look's good. great work